salt/modules/zypper.py

```diff
@@ -751,10 +751,7 @@ def list_pkgs(versions_as_list=False, **kwargs):
                                      python_shell=False,
                                      output_loglevel='trace')
         for line in output.splitlines():
-            pkginfo = salt.utils.pkg.rpm.parse_pkginfo(
-                line,
-                osarch=__grains__['osarch']
-            )
+            pkginfo = salt.utils.pkg.rpm.parse_pkginfo(line)
```

Comment:
I'm not sure if we can safely remove `osarch` here. Wouldn't it be better to check if `__grains__['osarch']` is `armv7l` and then adapt?

Reply:
`osarch` is redundant here, check `salt.utils.pkg.rpm` sources